component/openshift_template_renderer.libsonnet

```diff
@@ -0,0 +1,63 @@
+// OpenShift template renderer
```

Comment:
Note that the template renderer cannot be implemented following our Jsonnet style-guide as either `render_tpl_list` or `render_tpl_obj` will complain about an undefined reference to the other function if we define the functions as `local`s.

I'm open to move the renderer out of the component, if we think it is more generally useful.

Reply:
> I'm open to move the renderer out of the component, if we think it is more generally useful.

This might be usefull but I suggest to do this once we have an actual need.